src/widgets/storage/components/tab/tab.tsx

```diff
@@ -0,0 +1,59 @@
+import * as styles from './tab.css';
+
+export type TabLabelTypes = 'compare' | 'bookmark';
+
+export interface TabLabelProps {
+  value: TabLabelTypes;
+  onChange: (nextValue: TabLabelTypes) => void;
+  labels?: {
+    compare?: string;
+    bookmark?: string;
+  };
+}
+
+export const Tab = ({ value, onChange, labels }: TabLabelProps) => {
+  const compareLabel = labels?.compare ?? '폰트비교';
+  const bookmarkLabel = labels?.bookmark ?? '찜목록';
+
+  const handleCompareClick = () => {
+    if (value === 'compare') {
+      return;
+    }
+    onChange('compare');
+  };
+
+  const handleBookmarkClick = () => {
+    if (value === 'bookmark') {
+      return;
+    }
+    onChange('bookmark');
+  };
+
+  return (
+    <div className={styles.container}>
+      <button
+        type='button'
+        onClick={handleCompareClick}
+        className={styles.tabButton({
+          state: value === 'compare' ? 'active' : 'inactive',
+        })}
+        aria-pressed={value === 'compare'}
+      >
+        {compareLabel}
+      </button>
+
+      <span className={styles.divider} aria-hidden='true' />
+
+      <button
+        type='button'
+        onClick={handleBookmarkClick}
+        className={styles.tabButton({
+          state: value === 'bookmark' ? 'active' : 'inactive',
+        })}
+        aria-pressed={value === 'bookmark'}
+      >
+        {bookmarkLabel}
+      </button>
+    </div>
+  );
+};
```

Comment:
```suggestion
};

export default Tab;
```

이렇게 해주세요 ! 

나중에 이 컴포넌트를 import하는 방식이 달라져요 !!

나은님처럼 

```tsx
export const Tab = () => {};
```
이 방식으로 하면 나중에 import 하는 방식이
```tsx
import { Tab } from '@widgets/components/tab';
```

이렇게 되는데 이름이 다르면 오류가 나거나, alias를 따로 설정해줘야 해요 !

```tsx
const Tab = () => {};
export default Tab;
```

이렇게 하면 가져올 때 중괄호 없이 import할 수 있고,
원한다면 이름을 자유롭게 바꿔서 사용할 수도 있어요 !

default export는 파일의 **대표 컴포넌트**라는 의도를 더 명확히 보여줘요.
import 구문이 간결해지고 컴포넌트 파일마다 하나의 주 기능만 있을 때 직관적이랍니다.

페이지 컴포넌트 단위에서는 default export가 더 자연스러운 경우가 많아요 !